components/openzwave-adapter/src/lib.rs

```diff
@@ -173,6 +173,21 @@ fn set_ozw_vid_from_taxo_value(vid: &ValueID, value: Value) -> Result<(), TaxoEr
     result.map_err(|e| TaxoError::InternalError(InternalError::GenericError(format!("Error while setting a value: {}", e))))
 }
 
+fn start_pairing(ozw: &ZWaveManager, home_id: u32, value: &Value) -> Result<(), TaxoError> {
+    match *value {
+        Value::Secure(ref is_secure) => {
+            let is_secure_bool = match *is_secure { Secure::Insecure => false, Secure::Secure => true };
+            try!(
+                ozw.add_node(home_id, is_secure_bool)
```

Comment:
Yeah - I tested this and instead of passing is_secure_bool, you can just put: *is_secure == Secure::Secure